app/main.py

```diff
@@ -1 +1,5 @@
-#  write your code here
+from app.animal import Animal, Cat, Dog
+
+
+def feed_animals(ls_animals: list[Animal | Cat | Dog]) -> int:
+    return sum([animal.feed() for animal in ls_animals])
```

Comment:
```suggestion
    return sum(animal.feed() for animal in ls_animals)
```
Converting to list is not needed